install_config/persistent_storage/dynamically_provisioning_pvs.adoc

```diff
@@ -0,0 +1,160 @@
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+You can provision your OpenShift cluster with storage dynamically when running
+in a cloud environment. The Kubernetes
+link:../../architecture/additional_concepts/storage.html[persistent volume]
+framework allows administrators to provision a cluster with persistent storage
+and gives users a way to request those resources without having any knowledge of
+the underlying infrastructure.
+
+Many storage types are available for use as persistent volumes in OpenShift.
+While all of them can be statically provisioned by an administrator, some types
+of storage can be created dynamically using an API. These types of storage can
+be provisioned in an OpenShift cluster using the new and experimental dynamic
+storage feature.
+
+[IMPORTANT]
+====
+Dynamic provisioning is currently a Technology Preview feature, introduced in
+OpenShift Enterprise 3.1.1. The feature is experimental and expected to change
+in the future as it matures and feedback is received from users. New ways to
+provision the cluster are planned and the means by which one accesses this
+feature is going to change. Backwards compatibility is not guaranteed.
+====
+
+[[dynamic-pvs-provisioner-plugins]]
+== Available Provisioner Plug-ins
+
+Several OpenShift plug-ins have generic implementations for dynamic provisioning
+that use a provider's API to create new storage resources. The following
+provisioner plug-ins are currently available:
+
+[options="header"]
+|===
+
+|Storage Type |Provisioner Plug-in Name |Required Cloud Configuration
+
+|OpenStack Cinder
+|`kubernetes.io/cinder`
+|link:../../install_config/configuring_openstack.html[Configuring for OpenStack]
+
+|AWS Elastic Block Store (EBS)
+|`kubernetes.io/aws-ebs`
+|link:../../install_config/configuring_aws.html[Configuring for AWS]
+
+|GCE Persistent Disk (gcePD)
+|`kubernetes.io/gce-pd`
+|link:../../install_config/configuring_gce.html[Configuring for GCE]
+|===
+
+[IMPORTANT]
+====
+For any chosen provisioner plug-ins, configuration for the relevant cloud
+infrastructure must be set up as well, per *Required Cloud Configuration* in the
+above table.
+====
+
+Future provisioner plug-ins will include the many types of storage a single
+provider offers. AWS, for example, has several types of EBS volumes to offer,
+each with its own performance characteristics. They also have an NFS-like
+storage option. More provisioner plug-ins will be implemented for the supported
+storage types available in OpenShift.
+
+[[dynamic-pvs-configuring-storage-classes]]
+== Configuring Storage Classes
+
+Cluster administrators can configure their clusters to use dynamic provisioning
+for persistent volumes by defining _storage classes_ in the master configuration
+file (the *_/etc/origin/master/master-config.yml_* file by default). A storage
+class is a composite of the class name and the name of the provisioner plug-in,
+separated by a slash:
```

Comment:
This is no longer necessary.  There is now 1 implied provisioner per cloud without configuration.

In other words, you automatically get an EBS provisioner when OpenShift is correctly configured to run in AWS.